src/dbs/mod.rs

```diff
@@ -82,19 +86,39 @@ Targets must be in the form of <host>[:<port>], <ipv4|ipv6>[/<mask>]. For exampl
 	#[arg(value_parser = super::cli::validator::net_targets)]
 	allow_net: Option<Targets<NetTarget>>,
 
+	#[arg(
+		help = "Allow all RPC methods to be called. Optionally, you can provide a comma-separated list of RPC methods to allow. This is a server capability"
+	)]
+	#[arg(env = "SURREAL_CAPS_ALLOW_RPC", long, conflicts_with = "allow_all")]
```

Comment:
why can't you --allow-all and --allow-rpc? (and likewise for http)

Reply:
In the context of that PR you are right, as these capabilities are not already included in `--allow-all`. However, after our discussion, they will be included and we probably want users to be aware of this fact when they use CLI arguments that are redundant.

Reply:
After the changes were made in #4763, we can remove this conflict. I have done so in [023d049](https://github.com/surrealdb/surrealdb/pull/4512/commits/023d0497c77f731e8c9727cb9703c3a985f55ffa).